### ISSUE TITLE
Changing user currency does not update prices

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#3833]
## Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
Changing user currency is not updating prices.
<img src="http://g.recordit.co/4NanbiZMWs.gif">

This is not an issue with reactivity, the prices stay in USD when refreshing as well.
It does not appear to be a permissions issue, as being logged in as an admin does not resolve this issue.

## Steps to Reproduce
1. Start the app
2. Change user currency to something other than USD

### Versions
```
Node: 8.9.0
NPM: 5.6.0
Meteor Node: 8.9.4
Meteor NPM: 5.6.0
Reaction CLI: 0.28.0
Reaction: 1.8.1
Reaction branch: master
Docker: 17.12.0-ce
```

**Comment by zenweasel** [comment]
Do you have openexhange rates configured and working correctly?

**Comment by zenweasel** [comment]
It looks like the `getRates` job is no longer running in the current version

**Comment by prinzdezibel** [comment]
Took me the better part of the day ..
I got off-track when I've been thinking, it must have to do with the changes from @mikemurray. But that seems not the case. Its just that Jobs that are scheduled via later.js don't run before the first scheduled time, no matter what. https://github.com/reactioncommerce/reaction-job-queue/blob/4bbc524713e55349a16430d014e68384fb1638a3/src/common/jobCollectionBaseFactory.js#L1096-L1102

**Comment by zenweasel** [comment]
I think this actually used to work because of a bug in the original job collection framework where all scheduled jobs would run on first start. I was running into this on the demo site where every time the container would be created it would reset the db 20 times in a row. But it was fixed in the latest version (which we never upgraded to)

**Comment by zenweasel** [comment]
Closed via #3835
